tests/syscall/fs/enc/file_system.h

```diff
@@ -416,6 +444,32 @@ class stream_file_system
         return ret;
     }
 
+    ssize_t pread(file_handle file, void* buf, size_t count, off_t offset)
+    {
+        const long previous_offset = ::ftell(file);
```

Comment:
It's probably worth commenting that this does not test 'pread()'/'pwrite()' respectively, but is only implemented to satisfy 'test_all()' being implemented such that all methods must pass per the enc.c expectations regardless of the implementation class.

@nonpolarity I suspect that the Windows implementation would likely have to adopt the seek and read/write approach as illustrated here. We might also want to consider restructuring the test suite so that it has the appropriate test granularity instead of requiring a lot of bogus methods.